src/lib/notifications.ts

```diff
@@ -2,5 +2,5 @@ import { SingleSessionAbacusNotificationTypes } from '@/constants/notifications'
 
 export const isAbacusNotificationSingleSession = (notificationId: string) => {
   const notificationType = notificationId.split(':')[0];
-  return SingleSessionAbacusNotificationTypes.includes(notificationType);
+  return SingleSessionAbacusNotificationTypes.includes(notificationType!);
```

Comment:
or change to `notificationType && SingleSessionAbacusNotificationTypes.includes(notificationType)`